core/database/api/repo_router.js

```diff
@@ -146,18 +175,16 @@ router.post('/create', function (req, res) {
     id: joi.string().required(),
     title: joi.string().required(),
     desc: joi.string().optional(),
-    domain: joi.string().optional(),
-    capacity: joi.number().integer().min(0).required(),
+    capacity: joi.number().integer().min(1).required(),
```

Comment:
What is the reason for this change? Are you just wanting to ensure that when someone creates a repo they actually assign something to it?

Reply:
are you referring to the capacity? if so, a repo must have a non-zero capacity to make sense. The original min(0) was a mistake.

Reply:
Makes sense.